### PR TITLE
feat!: remove Transifex calls for OEP-58

### PR DESCRIPTION
**Pull request by hopewise**
## Breaking change!

This change breaks the Jenkins transifex integration which has been deprecated in favor of the new GitHub Transifex App integration as part of [OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

## Changes

 - Removes all direct use of `tx pull` and `tx push` commands from the micro-frontend in favor
of the [`atlas pull`](https://github.com/openedx/openedx-atlas/) command.
 - Remove source and language translations from the repositories, hence no `.json` files will be committed into the repos. 
 - `src/i18n/index.js` should export and empty array so the `make pull_translations` override it with the dynamic list of languages.
 - Remove the experimental `OPENEDX_ATLAS_PULL` flag to make `atlas pull` the default. 
 - Remove all Transifex related `Makefile` targets and other files.

Merge timeline
-----------------------

This should only be merged after [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification) is fully implemented.

The timing announcement will be shared by @brian-smith-tcril on [#translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T) Open edX Slack channel.

Keep this pull request as a draft to prevent accidental merge.

### Pre-merge checklist

 - [ ] https://github.com/openedx/wg-translations/issues/20 is approved

References
----------

This contribution is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which is sparked by the [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

Up-to-date project overview and details are available in the [Approach Memo and Technical Discovery: Translations Infrastructure Implementation](https://docs.google.com/document/d/11dFBCnbdHiCEdZp3pZeHdeH8m7Glla-XbIin7cnIOzU/edit#) document.

Join the conversation on [Open edX Slack #translations-project-fc-0012](https://openedx.slack.com/archives/C04R6TUJB7T).

Check the links above for full information about the overall project.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @hopewise!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.




:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: Makefile [hunk 1]
```diff
@@ -29,50 +21,28 @@ requirements:  ## install ci requirements
 	npm ci
 
 i18n.extract:
-	# Pulling display strings from .jsx files into .json files...
-	rm -rf $(transifex_temp)
```
Comment (OmarIthawi) [inline Makefile:33]: Despite the name, I think this line is still needed.

**Review by OmarIthawi — COMMENTED** [review]
Thanks @hopewise! Please add back the three lines I've pointed out and this pull request should be ready to go.

This PR makes a great first contribution.

File: Makefile [hunk 2]
```diff
@@ -1,16 +1,10 @@
 npm-install-%: ## install specified % npm package
 	npm install $* --save-dev
 	git add package.json
-export TRANSIFEX_RESOURCE = frontend-app-gradebook
-transifex_langs = "ar,de,es_419,fa_IR,fr,fr_CA,hi,it,pt,ru,uk,zh_CN"
 
 intl_imports = ./node_modules/.bin/intl-imports.js
 transifex_utils = ./node_modules/.bin/transifex-utils.js
 i18n = ./src/i18n
-transifex_input = $(i18n)/transifex_input.json
```
Comment (OmarIthawi) [inline Makefile:10]: This line is also used by `make i18n.concat`. Please keep it.

File: Makefile [hunk 3]
```diff
@@ -29,50 +23,30 @@ requirements:  ## install ci requirements
 	npm ci
 
 i18n.extract:
-	# Pulling display strings from .jsx files into .json files...
+	# Pulling display strings from .jsx files into .json files...	
 	rm -rf $(transifex_temp)
 	npm run-script i18n_extract
-
+
```
Comment (OmarIthawi) [inline Makefile:29]: We've added few unneeded spaces. Would you mind undoing those?

**Review by OmarIthawi — APPROVED** [review]

**Comment by OmarIthawi** [comment]
Thanks @hopewise!

Confirming that this PR is ready to be merged as soon as https://github.com/openedx/wg-translations/issues/20 is given green light.

**Review by brian-smith-tcril — APPROVED** [review]

**Pull request merged**